### DISCUSSION TITLE
v2/guide/markdown/flowchart

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#144] [Announcements]
# v2/guide/markdown/flowchart

A vuepress theme with tons of features✨

https://vuepress-theme-hope.github.io/v2/guide/markdown/flowchart.html

**Comment by DjCodeGit** [comment]
Hello,
It's not work with the last version.

if i make a simple example (the same of the guide). I have a spinning circle on the page

**Reply by DjCodeGit** [reply]
Ok in production Mode, it's work!!!! :thinking:
